spec/models/lead_provider_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe LeadProvider, type: :model do
+  it "can be created" do
+    expect {
+      LeadProvider.create(name: "Test Lead Provider")
+    }.to change { LeadProvider.count }.by(1)
+  end
+
+  it "can have a partnership" do
+    lead_provider = LeadProvider.create!(name: "Test Lead Provider")
```

Comment:
I think for these boilerplate tests for associations or validations the baked in matchers are better: https://github.com/thoughtbot/shoulda-matchers#usage . I mean this sort of thing: describe 'associations' do it {is_expected.to have_many (:blah)} end etc.